theme/type/type.less

```diff
@@ -10,52 +10,60 @@
 @font-descender-height-scale: 0.1365;
 
 // Hierarchy
-@hero-type-row-span: 6;
 @hero-type-scale: 4.2;
-@hero-type-row-span-mobile: 5;
-@hero-type-scale-mobile: 3.6;
+@hero-type-row-span: 6;
```

Comment:
There's a much larger diff here because I wanted to re-order all the variables to follow a consistent pattern—otherwise this change was actually really hard to write, let alone read.